ui/app/src/services/publishing.ts

```diff
@@ -137,6 +137,14 @@ export function publishByCommits(
   }).pipe(map(() => true));
 }
 
+export function publishAll(siteId: string, publishingTarget: string, comment: string): Observable<true> {
+  return postJSON('/studio/api/2/publish/all', {
+    siteId,
+    publishingTarget
+    // comment // TODO: pending API update
```

Comment:
Backend fails if sent? If you can send it anyway best so we don't forget to uncomment and such

Reply:
I tried sending it anyway but it was failing, the API is updated now, so I'll update